redaxo/bin/release

```diff
@@ -92,6 +92,9 @@ class rex_release
 
     private function createChangelog()
     {
+        // avoid PREG_JIT_STACKLIMIT_ERROR, changelog parts can be long
```

Comment:
ich glaube wir sollten das ganz top level in diesem skript hier machen... wg. exceptions o.ä. gibt es "zu viele wege" wie diese methode hier enden könnte ohne dass der `pcre.jit` zuverlässig wieder aktiviert wird

.. daher einfach generell deaktivieren, wird ohnehin keinen impackt haben auf das ergebnis (und die dauer) dieses skripts, oder?